config/jobs/kubernetes-security/genjobs.go

```diff
@@ -215,6 +226,10 @@ func convertJobToSecurityJob(j *config.Presubmit, dropLabels sets.String) {
 		if needStagingFlag {
 			container.Args = append(container.Args, "--stage=gs://kubernetes-security-prow/ci/"+j.Name)
 		}
+		// GCP e2e use a fixed project for security testing
+		if isGCPe2e {
+			container.Args = append(container.Args, "--gcp-project=k8s-jkns-pr-gce-etcd3")
```

Comment:
This project is safe to use for these purposes?

Reply:
@cjwagner pre-boskos migration they were using this project already, and now no-one else is using this project. We probably should replace the service account in the project IAM though.